### PR TITLE
Add info on ratio meanings to section on chroma subsampling

### PR DESCRIPTION
**Pull request by KimBurgess**
Addresses #24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -211,16 +211,21 @@ Generally, **displays** (monitors, TVs, screens and etc) utilize **only the RGB
...
 
 How much should we reduce the chroma resolution?! It turns out that there are already some schemas that describe how to handle resolution and the merge (`final color = Y + Cb + Cr`).
 
-These schemas are known as subsampling systems (or ratios), they are identified by the numbers: **4:4:4, 4:2:3, 4:2:2, 4:2:1, 4:1:1, 4:2:0, 4:1:0 and 3:1:1**. And each one of them defines how much should we discard in the chroma resolution as well as how we should merge the three planes (Y, Cb, Cr).
+These schemas are known as subsampling systems and are expressed as a 3 part ratio - `a:x:y` which defines the chroma resolution in relation to a `a x 2` block of luma pixels.
+
+ * `a` is the horizontal sampling reference (usually 4),
+ * `x` is the number of chroma samples in the first row of `a` pixels (horizontal resolution in relation to `a`), and
+ * `y` is is the numer of changes of chroma samples between the first and seconds rows of `a` pixels.
+
```
Comment (leandromoreira) [inline README.md]: I think that it's valid also to report that `This notation is not valid for all combinations and has exception` according to [wikipedia](https://en.wikipedia.org/wiki/Chroma_subsampling#Sampling_systems_and_ratios).

File: README.md [hunk 2]
```diff
@@ -211,16 +211,21 @@ Generally, **displays** (monitors, TVs, screens and etc) utilize **only the RGB
 
 ### Chroma subsampling
 
-Once we are able to separate luma from chroma, we can take advantage of the human visual system that is more capable of seeing luma than chroma. **Chroma subsampling** is the technique of encoding images using **less resolution for chroma than for luma**.
-
+With the image represented as luma and chroma components, we can take advantage of the human visual system's greater sensitity for luma resolution rather than chroma to selectively remove information. **Chroma subsampling** is the technique of encoding images using **less resolution for chroma than for luma**.
```
Comment (leandromoreira) [inline README.md]: 🥇 👏 👏 👏 👏 👏

File: README.md (hunk 1 above)
Comment (KimBurgess) [inline README.md]: Good catch. 4:1:0 is based on a `4 x 4` luma block with a single chroma sample (for both Cr and Cb), but I can't find any information on *why* this exception exists.

Comment (leandromoreira) [inline README.md]: No, 😞

**Comment by leandromoreira** [comment]
Thank you very much @KimBurgess 🍰

**Comment by KimBurgess** [comment]
No problems!

**Comment by KimBurgess** [comment]
It was bugging me that I couldn't find further info. It looks like the information on that article is not correct. It is 4:4:1 that is a special case. There is some further information in the section on 'irregular notation' here: http://dougkerr.net/Pumpkin/articles/Subsampling.pdf#page=7.

I'll update and shoot you another PR later today.

**Pull request merged**